src/frontend/overlay/src/components/organisms/status/Tickers.tsx

```diff
@@ -26,13 +26,14 @@ type JSTickerBodyProps = { pos: number };
 
 const JSTickerBody = styled.div.attrs<JSTickerBodyProps>(({ pos }) => ({
     style: {
-        left: pos,
+        transform: `translate3d(${pos}px, 0, 0)`,
```

Comment:
I would leave a comment about why you chose translate3d here, and maybe add https://jsperf.app/translate3d-vs-xy